recipes/strainy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.1" %}
+{% set name = "strainy" %}
+
+
+package:
+   name: strainy
+   version: {{ version }}
+
+
+source:
+   url: https://github.com/katerinakazantseva/strainy/archive/refs/tags/{{ version }}.tar.gz
+   sha256: bdbac3961fa840344ac67e249de8f6d4114e0796c5b40bb5535d8e59f4dc63d1
+
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+  run_exports:
+    - {{ pin_subpackage("strainy", max_pin="x") }}  
+
+
+requirements:
+  host:
+  - python ==3.10
```

Comment:
Why do you need this pin when it is a noarch package?

Reply:
the lint check didn't work until I added the pin.

please see details here
https://github.com/bioconda/bioconda-recipes/pull/50356/checks?check_run_id=29666772372

Reply:
`bioconda_utils.lint.check_build_help.missing_run_exports`  was about https://github.com/bioconda/bioconda-recipes/pull/50356/files#diff-8b101e069732d9958025a69406acf8c5cbc74923b68871aceb74f7caba0fcd72R19-R20

Reply:
exactly, was that mentioned in the first post? or was it about the host  requirements (python)?

Reply:
what is the `first post` ?

Reply:
`Why do you need this pin when it is a noarch package?`

Reply:
There seems to be a misunderstanding here, can you tell me what exactly should be removed? As I wrote above `run_exports` cannot be deleted because in this case the lint check fails

Reply:
https://github.com/bioconda/bioconda-recipes/pull/50356/commits/5aa787500e9fd95e996f03282fb48173742b4d69